source/docs/software/hardware-apis/sensors/index.rst

```diff
@@ -1,6 +1,8 @@
 Sensors
 =======
 
+Sensors are an integral way of having your robot hardware and software communicate with each other. This section highlights interfacing with those sensors at a software level.
+
 .. toctree::
```

Comment:
It seems odd that only some of the items in the TOC say -software. It should probably be all or none

Reply:
This is because a duplicate article exists in hardware. We can't have articles with the same name.